.github/workflows/release.yml

```diff
@@ -324,3 +327,94 @@ jobs:
 
           gh pr comment $pr_number --body-file comment-body.txt --edit-last
 
+  # -----------------------------------------------------------------------------------
+  # Build rerun_c library binaries:
+
+  build-rerun_c-and-upload-linux:
+    needs: [checks]
```

Comment:
These should all depend on `version`

Reply:
I'm not even sure why it works right now, does it use the right version?

Reply:
```
❯ pip install rerun-sdk==0.10a8
…
❯ python -m rerun --version
rerun_py 0.10.0-alpha.8 …
```

it did, but perhaps by luck